keyboards/gmmk/pro/ansi/keymaps/mattgauf/helpers.c

```diff
@@ -0,0 +1,41 @@
+/* Copyright 2021 Matt Gauf
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "rgb_matrix.h"
+
+void rgb_matrix_set_color_underglow(uint8_t red, uint8_t green, uint8_t blue) {
+    for (uint8_t ii = 0; ii < DRIVER_LED_TOTAL; ii++) {
+        if (g_led_config.flags[ii] & LED_FLAG_UNDERGLOW) {
+            rgb_matrix_set_color(ii, red, green, blue);
+        }
+    }
+}
```

Comment:
This may be better/simpler: 
```c
void rgb_matrix_set_color_flag(uint8_t red, uint8_t green, uint8_t blue, uint8_t flags) {
    for (uint8_t ii = 0; ii < DRIVER_LED_TOTAL; ii++) {
        if (g_led_config.flags[ii] & flags) {
            rgb_matrix_set_color(ii, red, green, blue);
        }
    }
}
```

Reply:
Those functions aren't being used right now, but I like the suggestion. I'm refactoring it now, and I'll update this PR.